project/src/main/webapp/scripts/groups.js

```diff
@@ -13,20 +13,93 @@ function drop(ev) {
   ev.target.appendChild(document.getElementById(data));
 }
 
+async function addAnswers() {
+    var servletURL = "/addAnswers?assignment-key=" + getUrlVars()['assignment-key'] + "&question-key=" + getUrlVars()["question-key"];
+    const response = await fetch(servletURL);
+    const answers = await response.json(); 
+    console.log(answers)
+}
+
 async function getAnswers() {
     var servletURL = "/answer?assignment-key=" + getUrlVars()['assignment-key'] + "&question-key=" + getUrlVars()["question-key"];
 
     const response = await fetch(servletURL);
     const answers = await response.json(); 
-    // show them on the page
 
+    var myDiv = document.getElementById("group-container");
+
+    let groups = new Map();
+
+    //create group => answers hash
+    answers.forEach(answer => {
+        var filePath = "../" + answer.filePath;
+        var groupKey = answer.groupKey;
+        
+        var image = document.createElement("img");
+        image.src = "../" + filePath;
+        image.setAttribute("overflow", "hidden");
```

Comment:
I think typically you'll want to set the element's class and specify attributes in css rather than specifying attributes manually here